app/services/omniauth_authorizer.rb

```diff
@@ -44,4 +46,8 @@ def update_auth
   def update_session
     session[:omniauthed] = true
   end
+
+  def user
```

Comment:
Because using a local would make the `perform` method have too many statements. I renamed the method to `auth_user`.